website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -189,3 +189,9 @@ move %USERPROFILE%\Downloads\dbt_cloud.yml %USERPROFILE%\.dbt\dbt_cloud.yml
 This command moves the `dbt_cloud.yml` from the `Downloads` folder to the `.dbt` folder. If your `dbt_cloud.yml` file is located elsewhere, adjust the path accordingly.
 
 </Expandable>
+
+<Expandable alt_header="How to skip artifacts from being downloaded">
+
+By default, all artifacts are downloaded when you execute dbt commands from the dbt Cloud CLI. To skip these files from being downloaded, add `--download-artifacts=false` to the dbt command you want to run. This can help improve run-time performance but can also result in unexpected behavior so be careful. 
```

Comment:
"This can help improve run-time performance but may break workflows that depend on assets like the manifest." Ideally with a link to the manifest/artifacts docs.